include/class-email-log-list-table.php

```diff
@@ -245,8 +249,10 @@ function process_bulk_action() {
 					$selected_ids = $ids;
 				}
 
-				// Can't use wpdb->prepare for the below query. If used it results in this bug
```

Comment:
This may be a personal preference, but if the comment is only 2-3 lines, then just let's use only `//` This will save us 2 lines and will make it more readable.